demo/src/screens/MenuStructure.js

```diff
@@ -175,10 +172,9 @@ export const navigationData = {
     title: 'Incubator (Experimental)',
     screens: [
       {title: 'Calendar', tags: 'calendar', screen: 'unicorn.components.IncubatorCalendarScreen'},
-      {title: 'ChipsInput (New)', tags: 'chips input', screen: 'unicorn.components.IncubatorChipsInputScreen'},
+      {title: 'ChipsInput', tags: 'chips input', screen: 'unicorn.components.ChipsInputScreen'},
```

Comment:
Should move under 'form'

Reply:
Why is related to this PR? I didn't change there anything

Reply:
hmm I think it was because the PR was targeting master and not v7